user/TASK/shoot_task/shoot_task.c

```diff
@@ -210,15 +210,31 @@ static void shoot_reversed_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *
 }
 
 
+static uint16_t shoot_count = 0;
+#define SHOOT_COUNT_MAX 1250
+#define SHOOT_COUNT_SWTICH 750
+
 /**
  * @brief Trigger motor and hopper motor continuously spinning to allow for rapid firing
  * @param Trigger motor and hopper motor structs
  * @retval None
  */
 static void shoot_rapid_control(Shoot_Motor_t *trigger_motor, Shoot_Motor_t *hopper_motor)
```

Comment:
Add comments like:
// push ball
// back track a bit to prevent jamming
// ...